.github/workflows/Docker-build-release.yaml

```diff
@@ -1,53 +1,33 @@
 ---
-name: Docker-build-release
+name: Update image in GHCR
+
+run-name: >
+  ${{
+    github.event_name == 'delete' && format(
+      'Delete `{0}{1}`',
+      github.event.ref_type == 'branch' && 'branch-' || '',
+      github.event.ref
+    )
+    || github.ref == 'refs/heads/main' && 'Update `dev`'
+      || format(
+        'Update `{0}{1}`',
+        !startsWith(github.ref, 'refs/tags') && 'branch-' || '',
+        github.ref_name
+      )
+  }} docker tag
 
 on:
   push:
-    branches: ['main']
+    branches-ignore: ['gh-pages']
     tags: ['v*']
-  release:
-    types: [published]
+  delete:
 
 jobs:
-  build-and-push-image:
+  push-or-delete-image:
     runs-on: ubuntu-latest
-    name: A job to build and push a docker image
+    name: Update GitHub Container Registry
     permissions:
       contents: read
       packages: write
-
     steps:
-      - name: Checkout repository
-        uses: actions/checkout@v3
-
-      - name: Read YAML
-        id: yaml-data
-        uses: jbutcher5/read-yaml@1.6
-        with:
-          file: metadata.yaml
-          key-path: '["image_name"]'
-
-      - name: Create tags
-        id: meta
-        uses: docker/metadata-action@v3
-        with:
-          flavor: |
-            latest=false
-          images: ghcr.io/uclahs-cds/${{ steps.yaml-data.outputs.data }}
-          tags: |
-            type=raw,enable=${{github.event_name == 'push'}},value=dev,event=branch
-            type=match,pattern=v(.*),group=1
-      - name: Log in to the Container registry
-        uses: docker/login-action@v1
-        with:
-          registry: ghcr.io/uclahs-cds
-          username: ${{ github.actor }}
-          password: ${{ secrets.GITHUB_TOKEN }}
-
-      - name: Build and push Docker image
-        uses: docker/build-push-action@v2
-        with:
-          context: .
-          push: true
-          tags: ${{ steps.meta.outputs.tags }}
-          file: ./docker/Dockerfile
+      - uses: uclahs-cds/tool-Docker-action@v2
```

Comment:
**comment (non-blocking)**: This works for now, but we may want to consider adding a parameter to this action to specify a custom path to the Dockerfile if necessary @nwiltsie

Reply:
That already exists with the `context` argument:

https://github.com/uclahs-cds/tool-Docker-action/blob/1f5dd8b37b40374f4a0089be14251971ceb48135/action.yml#L20-L22

Reply:
The context is different from specifying the path to the Dockerfile though no? Ex. the context is `.` but the Dockerfile can be in `./docker/Dockerfile` (this is the `file` argument for the same step that accepts the `context` argument)

Reply:
Fair, they're not quite the same thing. Let's add this as an issue on tool-Docker-action, it's not relevant to this PR.